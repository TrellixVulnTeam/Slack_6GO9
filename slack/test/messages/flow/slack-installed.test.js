const { SlackInstalled } = require('../../../lib/messages/flow');

describe('SlackInstalled', () => {
  test('works', async () => {
    const message = new SlackInstalled();
    expect(message.toJSON()).toMatchSnapshot();
  });
});
