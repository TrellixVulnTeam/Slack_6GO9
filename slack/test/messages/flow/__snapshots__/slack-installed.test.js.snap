// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SlackInstalled works 1`] = `
Object {
  "attachments": Array [
    Object {
      "color": "#24292f",
      "fallback": "You've successfully installed GitHub on this Slack workspace :tada:
To subscribe a channel to a repository use the following slash command:
/github subscribe owner/repository
",
      "text": "You've successfully installed GitHub on this Slack workspace :tada:
To subscribe a channel to a repository use the following slash command:
/github subscribe owner/repository
",
    },
    Object {
      "color": "#24292f",
      "fallback": "Looking for additional help? Try /github help",
      "text": "Looking for additional help? Try /github help",
    },
  ],
}
`;
